src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -1103,7 +1103,7 @@
             },
             "gridPos": {
                 "h": 8,
-                "w": 12,
+                "w": 8,
```

Comment:
needed?

Reply:
Might not be needed indeed.
I will spin everything up again so I can check

Reply:
just checking here